src/methods/delta/helpers/types.ts

```diff
@@ -73,7 +69,7 @@ type DeltaAuctionTransaction = {
   auctionId: string;
 };
 
-export type ParaswapDeltaAuction = {
+export type VeloraDeltaAuction = {
```

Comment:
since we're already making the name more abstract, I'd rather change it to simply `DeltaAuction`